packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -113,7 +165,32 @@ export class LibpodDockerode {
       });
     };
 
-    // add startPod
+    // add createPod
+    prototypeOfDockerode.createPod = function (podOptions: PodCreateOptions) {
+      const optsf = {
+        path: '/v4.2.0/libpod/pods/create',
```

Comment:
Why is version hard-coded? Should `4.2.0` be retrieve from a variable / somewhere else for when the user upgrades to `4.2.1`, etc?

Reply:
API is versioned so it should always work even if they change it in a more recent version
But there is also an open issue in podman 
https://github.com/containers/podman/issues/15376